docs/rackhd/api-usage-examples.rst

```diff
@@ -21,21 +21,26 @@ This example provides instructions on how to flash a BMC image on a Quanta (node
 2. Post the obm settings if they don't already exist for the node. An example on how to do this is shown in Section 7.1.8.1 here http://rackhd.readthedocs.io/en/latest/tutorials/vagrant.html#adding-a-sku-definition Section 7.1.8.1
 
 
-3. The firmware files need to be built into a SKU package
+3. Quanta-T41, Quanta-D51 Firmware upgrades:
+   Firmware upgrade process for these platforms is contained in the README.md file in their folders under on-skupack repositories.
+   Dell R730/R730XD, R630, C6320 Firmware upgrade(To be updated)
+
+4.  The firmware files need to be built into a SKU package
   
      **Build SKU Package**
      ::
      
-         $ ./build-package.bash <sku_pack_directory> <subname>
+         $ ./build-package.bash <sku_pack_name> <subname>
 
-   The new tarball is built in a folder called tarballs within the same directory 
+    <sku_pack_name> must be exactly the same with one the platforms listed on the root folder of on-skupack. E.g. it can be quanta-d51-1u, quanta-t41,dell-r630, etc. <subname>
+    can be any name you like. The new tarball is built in a folder called tarballs within the same directory
     
      ::
         
         $ /tarballs 
           sku_pack_directory_subname.tar.gz
           
-4. The SKU package that was built needs to be registered            
+5. The SKU package that was built needs to be registed
```

Comment:
Thanks Andrew for all the comments. I made a mistake and put the api-usage-example.rst to this PR, and this code change you commented  should be in this PR https://github.com/RackHD/docs/pull/354. The typo has been fixed.